lib/rules/spaced-comment.js

```diff
@@ -265,11 +295,37 @@ module.exports = {
         }
 
         /**
+         * Reports an ending spacing error with an appropriate message.
+         * @param {ASTNode} node - A comment node to check.
+         * @param {string} message - An error message to report.
+         * @param {string} match - An array of the matched whitespace characters.
+         * @returns {void}
+         */
+        function reportEnd(node, message, match) {
+
```

Comment:
Not a blocker, but it doesn't look like the other functions in this file pad the beginning of the block with an empty newline